palindrome/binary-search/src/Dictionary.java

```diff
@@ -0,0 +1,82 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Created by melmo on 1/3/17.
+ */
+public class Dictionary {
+    private List<String> dictionary = new ArrayList<>();
+
+
+    public boolean searchRecur(String word, double current, double min, double max){
```

Comment:
Overall this is very well done.  I am wondering why you chose to do doubles for these values since they are indexes.  I would have probably done the math by converting to doubles during the math of it instead of having them pass as doubles.  The biggest reason is that you technically pass a decimal to the method (0.5) and it only works because of your cast to an `int`